packages/vats/src/core/startWalletFactory.js

```diff
@@ -146,12 +146,7 @@ export const startWalletFactory = async (
       consume: { [Stable.symbol]: feeIssuerP },
     },
   },
-  {
-    options: {
-      perAccountInitialValue = (StableUnit * 25n) / 100n,
-      walletBridgeId = BRIDGE_ID.PROVISION_SMART_WALLET,
```

Comment:
are you sure this should be removed?

Reply:
lint complained that walletBridgeId is unused. How do I make it happy without removing it?

Reply:
reverted